packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -465,11 +460,7 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
                     (hasRemovedPureCssChunk && deps.size > 0)
                     ? `[${[...deps]
                         .map((d) =>
-                          JSON.stringify(
-                            relativePreloadUrls
-                              ? path.relative(path.dirname(file), d)
-                              : d
-                          )
+                          JSON.stringify(path.relative(path.dirname(file), d))
```

Comment:
Let's keep this code as it was previously, and same for 241. We may want to review in Vite 4 if we would like to avoid doing this "optimization" when base isn't relative.